projects/core/src/occ/adapters/organization/default-occ-organization-config.ts

```diff
@@ -12,6 +12,21 @@ export const defaultOccOrganizationConfig: OccConfig = {
           '/users/${userId}/orgUnitsAvailableApprovalProcesses',
         orgUnits: '/users/${userId}/orgUnits',
         orgUnit: '/users/${userId}/orgUnits/${orgUnitId}',
+        orgUnitUserGroups: '/users/${userId}/orgUnitUserGroups',
+        orgUnitUserGroup:
+          '/users/${userId}/orgUnitUserGroups/${orgUnitUserGroupUid}',
+        orgUnitUserGroupAvailableOrderApprovalPermissions:
+          '/users/${userId}/orgUnitUserGroups/${orgUnitUserGroupUid}/availableOrderApprovalPermissions',
+        orgUnitUserGroupAvailableOrgCustomers:
+          '/users/${userId}/orgUnitUserGroups/${orgUnitUserGroupUid}/availableOrgCustomers',
+        orgUnitUserGroupMembers:
+          '/users/${userId}/orgUnitUserGroups/${orgUnitUserGroupUid}/members',
+        orgUnitUserGroupMember:
+          '/users/${userId}/orgUnitUserGroups/${orgUnitUserGroupUid}/members/${orgCustomerId}',
+        orgUnitUserGroupOrderApprovalPermissions:
+          '/users/${userId}/orgUnitUserGroups/${orgUnitUserGroupUid}/orderApprovalPermissions',
+        orgUnitUserGroupOrderApprovalPermission:
+          '/users/${userId}/orgUnitUserGroups/${orgUnitUserGroupUid}/orderApprovalPermissions/${orderApprovalPermissionCode}',
```

Comment:
I think `userGroup` is enougth here as preffix

Reply:
I'd like to keep the full name to match the API call. It keeps it distinguished and provides the flexibility to add different types of "user groups" in the future.